### ISSUE TITLE
Fix category of babled-auto-gw-mode

### ISSUE THREAD

**Issue by aparcar** [libremesh/lime-packages#996]
It's in an entirely different category than (most) LibreMesh packages for no obvious reason

https://github.com/libremesh/lime-packages/blob/73e92cbb63e8d9ecb6660015e86b709b4b0ecacd/packages/babeld-auto-gw-mode/Makefile#L10

**Comment by PalAman-git** [comment]
I want to contribute can you please guide me how to fix it

**Comment by ilario** [comment]
Great!
Can you follow this guide? https://github.com/libremesh/lime-packages/blob/master/CONTRIBUTING.md
Then the only thing you should change is from:

```
  SECTION:=utils
  CATEGORY:=Utilities
```

to:
```
  SECTION:=lime
  CATEGORY:=LibreMesh
```

inside the `lime-packages/packages/babeld-auto-gw-mode/Makefile` file.
Thanks!

**Comment by PalAman-git** [comment]
I have fixed the issue and requested the pull request. please review and merge it

**Comment by ilario** [comment]
Fixed in #1006 
Thanks!
